tests/integration.rs

```diff
@@ -3558,15 +3563,18 @@ mod tests {
         }
 
         #[test]
-        #[cfg(target_arch = "x86_64")]
         fn test_serial_off() {
             let mut focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
             let guest = Guest::new(&mut focal);
             let mut workload_path = dirs::home_dir().unwrap();
             workload_path.push("workloads");
 
             let mut kernel_path = workload_path;
+
+            #[cfg(target_arch = "x86_64")]
             kernel_path.push("bzImage");
+            #[cfg(target_arch = "aarch64")]
+            kernel_path.push("Image");
```

Comment:
Can you do this instead: `let kernel_path = direct_kernel_boot_path();` ?`

Reply:
No problem.